packages/calcite-components/src/components/date-picker/date-picker.e2e.ts

```diff
@@ -28,6 +28,108 @@ describe("calcite-date-picker", () => {
     });
   });
 
+  describe("theme", () => {
+    const tokens = {
+      "--calcite-date-picker-border-color": {
+        targetProp: "borderColor",
+      },
+      "--calcite-date-picker-corner-radius": [
+        {
+          targetProp: "borderRadius",
+        },
+        {
+          shadowSelector: "calcite-date-picker-month-header",
+          targetProp: "--calcite-date-picker-month-header-corner-radius",
+        },
+      ] as TestSelectToken[],
```

Comment:
Do you still need to type cast after recent type updates to `themed`?

Reply:
yes

Reply:
Oh, I see why. Typecasting bypasses type checks at assignment time.

 If we export the type of `themed`'s `tokens` arg, we can type `tokens` here as 

```ts
const tokens: ThemedTokensArg /* example name 😅 */ = {
  // ...
};
```

Alternatively, we can pass the object directly to `themed` so that TS checks the type there.